packages/pyright-internal/src/languageService/symbolIndexer.ts

```diff
@@ -153,3 +154,14 @@ function collectSymbolIndexDataForName(
 
     indexSymbolData.push(data);
 }
+
+function shouldAliasBeIndexed(declaration: AliasDeclaration) {
+    if (
```

Comment:
The logic here is a little hard to follow for me. 

So an alias should be indexed if:
- ParseNodeType.ImportAs or
- ParseNodeType.ImportFromAs and
- It is defined

Would the true version of this condition make more sense when reading it?

Reply:
Yeah, I had a hard time following the logic here as well. Perhaps a comment would help clarify? It looks like you're proposing to include only those import statements that include an `as` clause (and therefore define an "alias" for the imported symbol)?

Reply:
Inverted the node type logic and added a comment.